pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -877,11 +877,11 @@ func verifyStatus(ctx *domainContext, status *types.DomainStatus) {
 		}
 		// check if qemu processes has crashed
 		hasQemu := status.VirtualizationMode == types.HVM || status.VirtualizationMode == types.FML || status.IsContainer
-		if configActivate && status.Activated && hasQemu && !hyper.IsDeviceModelAlive(status.DomainId) {
+		if err = hyper.CheckDeviceModelAlive(status.DomainId); configActivate && status.Activated && hasQemu && err != nil {
```

Comment:
Why call this when hasQemu is false, or configActivate is false? That's very odd logic

Reply:
Changed